.ci/run_nccl_test.sh

```diff
@@ -127,8 +127,9 @@ export LD_LIBRARY_PATH="${NCCL_DIR}/lib:${NCCL_RDMA_SHARP_PLUGINS_DIR}/lib:${LD_
     echo_hash_line
 
     MPIRUN_OPTIONS_SPECIFIC="\
-    -x NCCL_LL_THRESHOLD=0 \
-    -x NCCL_TREE_THRESHOLD=1000000000 \
+    -x NCCL_PROTO=Simple \
+    -x NCCL_ALGO=CollNet \
```

Comment:
`NCCL_ALGO=CollNet` is it not documented yet? Don't see such value in the official documentation https://docs.nvidia.com/deeplearning/sdk/nccl-developer-guide/docs/env.html#nccl-algo

Reply:
Yes, it's not documented cause nccl 2.5 with collnet support has not been released yet.